src/audio/rtnr/rtnr.c

```diff
@@ -384,10 +384,10 @@ static int rtnr_params(struct comp_dev *dev, struct sof_ipc_stream_params *param
 	cd->sink_stream.channels = audio_stream_get_channels(&sink_c->stream);
 
 	/* set source/sink stream overrun/underrun permitted */
-	cd->sources_stream[0].overrun_permitted = source_c->stream.overrun_permitted;
-	cd->sink_stream.overrun_permitted = sink_c->stream.overrun_permitted;
-	cd->sources_stream[0].underrun_permitted = source_c->stream.underrun_permitted;
-	cd->sink_stream.underrun_permitted = sink_c->stream.underrun_permitted;
+	cd->sources_stream[0].overrun_permitted = audio_stream_get_overrun(&source_c->stream);
```

Comment:
I wonder if it would make sense to rename the struct fields (with e.g. "_" prefix) at the same time as you make these changes. This would prevent any future accidental direct use. Or at least one would really have to try hard to use the struct fields directly.

Reply:
@lyakh But these will now only be used in the inlined functions in, so the ugliness is very well isolated. Nobody should be using or looking at these fields directly anymore, that's what  we want. But not blocking this PR on this.